### PR TITLE
Adds waddles to animated objects and various creatures

### PR DESCRIPTION
**Pull request by KoboldCommando**
# About The Pull Request

This adds a waddle animation to the animated items created by the staff of animation and the brand intelligence event.  It also gives them a heavy footstep sound.

It also adds a waddle to various animals that aren't too small and scuttley (lizards and roaches) or too large (cows and bears)

It purposefully DOESN'T add a waddle to old Ian since he's got mobility wheels.

## Why It's Good For The Game

I think they look a LOT nicer.  It gives a clear indication that the creature is moving around, and adds a little life to them.  In the future I hope to find something similar that will look good for the scuttling and large mobs.

https://user-images.githubusercontent.com/31165061/196010793-633a9d1b-0791-45b7-a6d4-fd678386d5b7.mp4

## Changelog

:cl:
add: Animated objects and several types of animals now look a bit more lively when moving around!
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm [hunk 1]
```diff
@@ -382,6 +385,7 @@
 /obj/item/udder/Initialize(mapload)
 	create_reagents(50)
 	reagents.add_reagent(/datum/reagent/consumable/milk, 20)
+	src.AddComponent(/datum/component/waddling)
```
Comment (nednaZ) [inline code/modules/mob/living/simple_animal/friendly/farm_animals.dm:388]: I don't think the udder itself needs the waddling component.
Probably just the cows.

**Review by nednaZ — CHANGES_REQUESTED** [review]
Looks just about fine, needs one change.

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm (hunk 1 above)
Comment (KoboldCommando) [reply code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: LMAO!  I was trying to make turkeys waddle and didn't look close enough at the path of the proc below it.

**Comment by KoboldCommando** [comment]
> Looks just about fine, needs one change.

Change made

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm [hunk 2]
```diff
@@ -376,6 +379,10 @@
 	gold_core_spawnable = FRIENDLY_SPAWN
 	chat_color = "#FFDC9B"
 
+/mob/living/simple_animal/chicken/turkey/Initialize(mapload)
+	. = ..()
+	src.AddComponent(/datum/component/waddling)
```
Comment (nednaZ) [inline code/modules/mob/living/simple_animal/friendly/farm_animals.dm:384]: I don't think this is actually needed.
The chicken parent call is already adding the waddling component

Comment (KoboldCommando) [reply code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: Fair point, fixed!

**Review by nednaZ — APPROVED** [review]
Looks good to me.

**Pull request merged**